airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```diff
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "ברירת מחדל תצוגת גרף",
+  "defaultToGridView": "ברירת מחדל תצוגת רשת",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
+    "assets": "משאבים (Assets)",
+    "browse": "עיין",
+    "admin": "אדמין",
+    "docs": "דוקומנטציה",
+    "plugins": "פלאגינים"
+  },
+  "browse": {
+    "auditLog": "לוג מערכת",
+    "xcoms": "(XComs)"
+  },
+  "admin": {
+    "Variables": "משתנים",
+    "Pools": "בריכת משאבים",
+    "Providers": "חבילות עזר",
+    "Plugins": "פלאגינים",
+    "Connections": "חיבורים",
+    "Config": "קונפיגורציות"
+  },
+  "timeRange": {
+    "duration": "משך זמן",
+    "lastHour": "השעה האחרונה",
+    "last12Hours": "12 השעות האחרונות",
+    "last24Hours": "24 השעות האחרונות",
+    "pastWeek": "השבוע החולף"
+  },
+  "docs": {
+    "documentation": "דוקומנטציה",
+    "githubRepo": "GitHub מאגר",
+    "restApiReference": "REST API רפרנס"
+  },
+  "states": {
+    "queued": "בהמתנה",
+    "running": "בריצה",
+    "success": "הצלחה",
+    "failed": "כשלון",
+    "skipped": "דולג",
+    "removed": "הוסר",
```

Comment:
הצליח \ נכשל, על משקל דולג \ הוסר

Reply:
That is not what it means... removed is when the task was removed from the dag and no longer present...